mdata/cwr_test.go

```diff
@@ -0,0 +1,53 @@
+package mdata
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/raintank/schema"
+)
+
+func TestArchiveRequestEncodingDecoding(t *testing.T) {
+	originalRequest := ArchiveRequest{
+		MetricData: schema.MetricData{
+			Name: "testMetricData",
+		},
+		ChunkWriteRequests: []ChunkWriteRequest{
+			{
+				TTL: 1,
+				T0:  2,
+			},
```

Comment:
this test would be more useful if it actually verified that the `.Data` is correct as well

Reply:
This test was broken and I didn't notice it. Apparently something weird happened when the cwr's `.Timestamp` was not set and got encoded/decoded. Once I've set it to any value the test passed.